### ISSUE TITLE
[BUG]The message in the notification list is missing

### ISSUE THREAD

**Issue by hhbisheng** [vitorpamplona/amethyst#912]
When encountering multiple replies, some replies are not displayed in "🔔", which leads to some replies being missed as a result. I have encountered this issue many times, and I have noticed that someone has replied to my messages under my post, but I cannot see any relevant notifications.
I am using the latest Google version of Amethyst with 0.87, which is strange because I should be able to find all the messages related to me in 🔔, but in reality, some messages are missing. I can read these messages, but they will not appear in my notification list.

**Comment by vitorpamplona** [comment]
There are a few things to check: 
1. If you are cited as a p-tag in that post. People can choose not to notify you if they wish.
2. Check if you are not filtering the Notifications screen by a list. The Top Bar must say Global instead of All Follows.
3.  Amethyst hides replies that don't reply directly to a post by you as author or as a thread that started by you. This is to remove notifications from threads that have taken a life on their own and people are just replying to each other.

**Comment by hhbisheng** [comment]
Maybe it's 1 or 3，
I have ruled out the second option, which may be related to their use of different clients, but I cannot be certain

**Comment by hhbisheng** [comment]
Now it has been confirmed to be the third scenario. Is this determined by the message mechanism of amethyst?
The other party also used amethyst, A replied to me, and then he replied to A, but I did not receive any notification of his message.

**Comment by vitorpamplona** [comment]
Correct. Further down messages do not notify you unless you are the root author
